### DISCUSSION TITLE
Ввод букв я, э, ё, ю только по немой клавишe `J`.

### DISCUSSION THREAD

**Discussion by dotland** [dotland/mnemonic-kb-ru#2] [Polls]
В текущей версии клавиатуры буквы *я*, *э*, *ё*, *ю* вводятся и́ по клавишу `J`, и́ по клавишу `Y`. Можно сделать клавишу `Y` не немой (обыкновенной), чтобы такие оканчания слов как *ые* были введены без пробелов.

**Comment by antonyveyre** [comment]
уже сделали :)

**Comment by dotland** [comment]
Исправлено в редакции v1.0.1.
